### PR TITLE
Testing Routing failure on browser

### PR DESCRIPTION
**Pull request by MaxAake**
Routing failure on testkit identified, associated with DNS caching on firefox when testing the browser driver.

Solved by starting firefox with a custom profile with no TTL for the DNS cache

### CODE REVIEW

**Comment by MaxAake** [comment]
[Investigate DNS failure in teamcity build](https://trello.com/c/ExrnN7hi/474-investigate-dns-failure-in-teamcity-build)

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ coverage
 *.code-workspace
 /testkit/CAs
 /testkit/CustomCAs
+/testkit/profile/*
```
Comment (bigmontz) [inline .gitignore]: nit: firefox_profile

File: testkit/profile/prefs.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+user_pref("network.dnsCacheExpiration", 0); // eslint-disable-line
+user_pref("network.dnsCacheExpirationGracePeriod", 0); // eslint-disable-line
```
Comment (bigmontz) [inline testkit/profile/prefs.js]:
```suggestion
// @eslint-disable
user_pref("network.dnsCacheExpiration", 0);
user_pref("network.dnsCacheExpirationGracePeriod", 0);
```

**Review by bigmontz — APPROVED** [review]
🍴

File: .gitignore (hunk 1 above)
Comment (bigmontz) [reply .gitignore]: forgot  to update .gitignore

File: testkit/backend.py [hunk 3]
```diff
@@ -34,7 +35,7 @@
             time.sleep(5)
             print("openning firefox")
             with open_proccess_in_driver_repo([
-                "firefox", "-headless", "http://localhost:8000"
+                "firefox", "-profile", "./testkit/firefox_profile", "-headless", "http://localhost:8000" # type: ignore
```
Comment (robsdedude) [inline testkit/backend.py]:
```suggestion
                "firefox", "--profile", "./testkit/firefox_profile", "--headless", "http://localhost:8000" # type: ignore
```
(nit)
I mean this works as is, but if you do `firefox --help`, it'll show the full-name options prefixed with `--`

**Review by robsdedude — APPROVED** [review]

**Pull request merged**